src/module.c

```diff
@@ -4278,6 +4285,16 @@ RedisModuleCallReply *RM_Call(RedisModuleCtx *ctx, const char *cmdname, const ch
         goto cleanup;
     }
 
+    /* Check if the user can run this command according to the current
+     * ACLs. */
+    int acl_errpos;
+    int acl_retval = ACLCheckAllPerm(c,&acl_errpos);
+    if (acl_retval != ACL_OK) {
+        addACLLogEntry(c,acl_retval,acl_errpos,NULL);
+        errno = EACCES;
```

Comment:
we need to document `EACCES` at the top, or even provide detailed specific errors for the various ACL errors (i think i rather not)
alternatively, maybe we wanna return an error reply? (i don't think so). @MeirShpilraien @guybe7?